lib/src/cmaf/ham/mapper/hls/mapHamToHls.ts

```diff
@@ -57,14 +58,18 @@ function _generateVideoManifestPiece(videoTrack: VideoTrack) {
 			return `#EXTINF:${segment.duration},${newline}${byteRange}${newline}${segment.url}`;
 		})
 		.join(newline);
-	const videoByteRange = videoTrack.byteRange != undefined ? `#EXT-X-BYTERANGE:${videoTrack.byteRange}${newline}` : '';
-	playlist = `#EXTM3U${newline}#EXT-X-TARGETDURATION:${videoTrack.duration}${newline}#EXT-X-MAP:URI="${videoTrack.urlInitialization}",${videoByteRange}${newline}${playlist}`;
+	const videoByteRange =
+		videoTrack.byteRange != undefined
```

Comment:
Double equal (`==` and `!=`) should never be used due to unexpected behaviours.
You can use just `videoTrack.byteRange` as condition instead. It will be true in the case `videoTrack.byteRange` is not `undefined` or `''`
```suggestion
		videoTrack.byteRange
```

same comment for line 84